lib/screens/screen.dart

```diff
@@ -1,22 +1,38 @@
 import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
 import 'package:flutter/widgets.dart';
 import 'package:flutter/foundation.dart';
 
 class Screen extends StatelessWidget {
   final Widget body;
-  Screen({this.body});
+  final SystemUiOverlayStyle statusBarStyle;
+
+  const Screen({
+    required this.body,
+    this.statusBarStyle = SystemUiOverlayStyle.light,
+  });
 
   @override
   Widget build(BuildContext context) {
-    if (defaultTargetPlatform == TargetPlatform.android) {
-      return Scaffold(
-        body: body,
-      );
+    Widget? child;
+
+    switch (defaultTargetPlatform) {
+      case TargetPlatform.android:
+        child = Scaffold(
+          body: body,
+        );
+        break;
+
```

Comment:
I think we should not add break line between `case` / `default` in `switch` block 🦖